Public/Sdk/SelfHost/BuildXL/Testing/QTest/qtestFramework.dsc

```diff
@@ -138,6 +145,8 @@ function runTest(args : TestRunArguments) : File[] {
         tags: args.tags,
         weight: args.weight,
         privilegeLevel: args.privilegeLevel,
+        coverageLogsDir: coverageLogDir,
+        qTestBuildType: qualifier.configuration === "release" ? "retail" : qualifier.configuration

```

Comment:
This file is only used for BuildXL selfhost build. if you get the qTestBuildType from here, it won't work for office build.